apps/src/applab/designElements/ThemeDropdown.jsx

```diff
@@ -73,6 +81,7 @@ export default class ThemeDropdown extends React.Component {
           onChange={this.handleChange}
           options={renderedOptions}
           placeholder={''}
+          clearable={false}
```

Comment:
i think it's fine to make this not clearable, but responding to your comment: "Remove clear icon (which didn't do anything anyways)" -- the clear icon set the theme back to "default" so it probably looked like it didn't do anything if you hadn't changed the current theme, but it was working as expected

Reply:
oh that makes sense, I'll go ahead and remove that from the description!